src/cogent3/app/__init__.py

```diff
@@ -155,9 +155,7 @@ def _get_app_matching_name(name: str):
         raise NameError(f"no app matching name {name!r}")
     elif table.shape[0] > 1:
         raise NameError(f"too many apps matching name {name!r},\n{table}")
-    modname, _ = table.to_list(columns=["module", "name"])[0]
-    mod = importlib.import_module(modname)
-    return getattr(mod, name)
+    return apps()[name].plugin
```

Comment:
if this generates a `KeyError` when `name` is incorrect, I think we might want to trap that call and raise a `ValueError` instead, with a custom message making it clear users should check for typos